code/modules/mapfluff/ruins/spaceruin_code/TheDerelict.dm

```diff
@@ -175,3 +175,96 @@
 	icon = 'icons/obj/turrets.dmi'
 	icon_state = "turretCover"
 	density = TRUE
+
+///Captain's log
+/obj/item/tape/captains_log
+	name = "captain's log"
+	desc = "A dusty old tape."
+	icon_state = "tape_blue"
+	used_capacity = max_capacity  // so the tape is full
+	storedinfo = list( // the captain recorded this in several segements
+		// 1st monologue (start of shift)
+		"\[00:02\] Every single day I spend on this station I become more and more certain that this was some kind of macabre punishment assignment.",
```

Comment:
this should include a comment or something indicating the source of this monologue, plus a credit to it's creator

Reply:
see desc of https://www.youtube.com/watch?v=7M-JPH5SOmI

Reply:
Applied with https://github.com/tgstation/tgstation/pull/68411/commits/fe3be539088add90f2d55c6f124a50f8cf8b8107